client/source/components/questionsAnswers/HelpfulA.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import * as requests from '../../utilities/axiosRequests.js';
+import { ButtonContainer } from './styles/Containers.styled';
+
+const HelpfulA = function CreateHelpfulAComponent({ id, getQuestions, helpfulness }) {
+  const [isReported, setIsReported] = useState(false);
+
+  const handleHelpfulA = function handleHelpfulA(event) {
+    event.target.setAttribute('disabled', true);
+    requests
+      .put(`/qa/answers/${id}/helpful`)
+      .then((results) => {
+        console.info(results.status);
+        getQuestions();
+      })
+      .catch((err) => console.error('handleHelpfulA error: ', err));
+  };
+
+  const handleReportedA = function handleReportedA(event) {
+    setIsReported(true);
+    event.target.setAttribute('disabled', true);
+    requests
+      .put(`/qa/answers/${id}/report`)
+      .then((results) => {
+        console.info(results.status);
+      })
+      .catch((err) => {
+        console.error('handleReportedA error: ', err);
+        setIsReported(false);
+      });
+  };
+
+  return (
+    <ButtonContainer>
+      <span>Helpful? </span>
+      <span>
+        <button
+          type="button"
+          onClick={handleHelpfulA}
+        >Yes {`(${helpfulness})`}
+        </button>
+        <span>  |  </span>
+        <button
+          type="button"
+          onClick={handleReportedA}
+        >{isReported ? 'Reported' : 'Report'}
```

Comment:
No action, just a comment. This is great - I should probably add the same to indicate it has already been clicked, and therefore reported.

Reply:
Thanks! That one is in the BRD for me

Reply:
Ah, I see it! I will align mine accordingly, as it doesn't seem to specify the reported features for the Review Tiles.